tests/storage/cbt/utils.py

```
@@ -0,0 +1,95 @@
+from utilities.constants.timeouts import TIMEOUT_5SEC, TIMEOUT_10MIN
+
+if TYPE_CHECKING:
+    from kubernetes.dynamic import DynamicClient
+    from ocp_resources.virtual_machine import VirtualMachine
+    from ocp_resources.virtual_machine_backup import VirtualMachineBackup
+
+LOGGER = logging.getLogger(__name__)
+
+CBT_TEST_DATA: str = "cbt-backup-test-data-content"
+CBT_INCREMENTAL_TEST_DATA: str = "cbt-incremental-backup-test-data"
+CBT_BOOT_DISK_TEST_DATA_FILE: str = "/tmp/cbt-test-data.txt"
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/cbt-test-data.txt"

(S108)